content/docs/start/visualization.md

```diff
@@ -0,0 +1,100 @@
+# Visualization with DVC Plots
+
+In parallel with versioning parameters and metrics by associating them with
```

Comment:
Early to edit language, but this paragraph is too complicated for the intro. 

- "multivariate outputs"
- DVC allows to generate plots - I think DVC also captures them, right? (keeps track)
- `In parallel with versioning parameters and metrics` - this means that it should go right after experiments show/run?